.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -16,6 +13,46 @@ on:
         required: true
         default: true
         type: boolean
+      owner-subject:
+        description: 'Owner (used for creating tmp billing project)'
+        required: true
+        default: 'hermione.owner@quality.firecloud.org'
+        type: string
+      non-owner-subject:
+        description: 'Student (non-owner used for e2e testing)'
+        required: true
+        default: 'harry.potter@quality.firecloud.org'
+        type: string
+      service-account:
+        description: 'Email address or unique identifier of the Google Cloud service account for which to generate credentials'
+        required: true
+        default: 'firecloud-qa@broad-dsde-qa.iam.gserviceaccount.com'
+        type: string
+      access-token-scopes:
+        description: 'List of OAuth 2.0 access scopes to be included in the generated token'
+        required: true
+        default: 'profile, email, openid'
+        type: string
+      tenant-id:
+        description: 'Azure tenant ID'
+        required: true
+        default: 'fad90753-2022-4456-9b0a-c7e5b934e408'
+        type: string
+      subscription-id:
+        description: 'Azure subscription ID'
+        required: true
+        default: 'f557c728-871d-408c-a28b-eb6b2141a087'
```

Comment:
can you add a comment on which subscription this is?

Reply:
Updated comment.